packages/authenticator/amplify_authenticator_test/lib/src/mock_authenticator_app.dart

```diff
@@ -1,17 +1,28 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-// ignore_for_file: diagnostic_describe_all_properties
+// ignore_for_file: diagnostic_describe_all_properties, invalid_use_of_visible_for_testing_member, implementation_imports
 
 import 'package:amplify_authenticator/amplify_authenticator.dart';
+import 'package:amplify_authenticator/src/blocs/auth/auth_bloc.dart';
+import 'package:amplify_authenticator/src/services/amplify_auth_service.dart';
+import 'package:amplify_authenticator_test/amplify_authenticator_test.dart';
 import 'package:amplify_authenticator_test/src/configs/email_config.dart';
-import 'package:amplify_authenticator_test/src/finders/authenticated_app_finder.dart';
-import 'package:amplify_authenticator_test/src/finders/authenticator_finder.dart';
 import 'package:amplify_flutter/amplify_flutter.dart';
 import 'package:amplify_integration_test/amplify_integration_test.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
 
+/// Use for testing the [MockAuthenticatorApp] widget locally.
+void main() {
```

Comment:
if you would like to commit this, can you move it to an /example dir and add all the platforms (not just iOS)